incompatibility/incompatibility-report/reporter.go

```diff
@@ -60,85 +60,86 @@ func DocumentBaseSearch(doc *openapiv3.Document) []*Incompatibility {
 // PathsSearch is a reporter that scans for incompatibilities in the paths component of an OpenAPI doc
 func PathsSearch(doc *openapiv3.Document) []*Incompatibility {
 	var incompatibilities []*Incompatibility
-	pathsKey := []string{"paths"}
```

Comment:
This would have been a good change to make stand alone.  Rename of methods or file renames are great thing to isolate.   As it is now, it looks like you renamed and changed variable names and changed logic.  Making it a bit harder to review.

Reply:
will try to better on this front!